src/Apps/ViewingRoom/ViewingRoomApp.tsx

```diff
@@ -26,7 +25,6 @@ const ViewingRoomApp: React.FC<ViewingRoomAppProps> = ({
 
   const isPreviewable =
     user &&
```

Comment:
You don't even _really_ need the `user` part... but I think it's harmless to leave in. This should unlock admins previewing VR's.